crates/l2/operator/block_producer.rs

```diff
@@ -13,12 +14,12 @@ use super::consensus_mock::ConsensusMock;
 pub async fn start_block_producer() {
     // This is the genesis block hash
     let mut current_block_hash =
-        H256::from_str("0x676fb5bffb4b4962edb2e1e03ac733597d5ba6ac290b230a2bf01448decae584")
+        H256::from_str("0x493dba6364c3e0b575b81efe8b255eb76d8fcd302517557beac9ead7816ea7a3")
             .unwrap();
 
     loop {
         let secret = Bytes::from_static(include_bytes!(
-            "/Users/ivanlitteri/Repositories/lambdaclass/ethereum_rust/jwt.hex"
+            "/Users/manu/Documents/Lambda/eth/ethereum_rust/jwt.hex"
```

Comment:
I'm guessing this is not supposed to be hardcoded

Reply:
It is not! This will be fixed before [ethereum_rust_l2_consensus_mock](https://github.com/lambdaclass/ethereum_rust/tree/ethereum_rust_l2_consensus_mock) is merged